schemas/development/xarf_shared.schema.json

```diff
@@ -341,12 +380,39 @@
           "minimum": 0
         },
         "FileHash": {
-          "type": "string",
-          "description": "hash value of the file"
+          "$ref": "#/properties/Hash"
         }
       },
       "required": ["FileName"]
     },
+    "Password": {
+      "description": "Password information",
+      "type": "object",
+      "properties": {
+        "PasswordHash": {
+          "$ref": "#/properties/Hash"
+        }
+      }
+    },
+    "Hash": {
+      "description": "The hash value of eg. a file or a password.",
+      "type": "object",
+      "properties": {
+        "HashValue": {
+          "type": "string"
+        },
+        "HashAlgorithm": {
+          "type": "string",
+          "enum": ["sha1", "sha2", "sha3", "sha256", "sha512", "md5"]
```

Comment:
other common algorithms like `bcrypt`, `scrypt`, `pbkdf2`, ...?
https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html

Reply:
Just saw your changes - I would use lowercase for `PBKDF2` too, just to be consistent. (otherwise, we would also have to change the `SHA` namings, for example, which are usually written in capital letters too)